_DeploymentAndDistroPackaging/ansible/doc/development.md

```diff
@@ -1,8 +1,15 @@
 # Development Mode
-This playbooks installs ciao from the latest ciao packaged for your distro.
+This playbooks are though to install ciao on production environments from the
```

Comment:
This sentence doesn't make sense.  How about

'By default, these playbooks install ciao on ..'